tests/dummy/app/components/routable-todo-message.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import stateFor from 'ember-state-services/state-for';
+
+export default Ember.Component.extend({
+  data: stateFor('todos', { key: 'todo.id' }),
```

Comment:
here it is `key: 'todo.id' above it is merely the key value, are both supported ?